### PR TITLE
Restrict access to staging hubs and imagebuilding-demo hub

### PR DESCRIPTION
**Pull request by consideRatio**
Closes https://github.com/2i2c-org/meta/issues/682

### CODE REVIEW

File: config/clusters/2i2c-aws-us/cosmicds.values.yaml [hunk 1]
```diff
@@ -82,20 +82,9 @@ jupyterhub:
         oauth_callback_url: https://cosmicds.2i2c.cloud/hub/oauth_callback
         shown_idps:
           - http://github.com/login/oauth/authorize
-          # Temporarily enable google & microsoft auth, to be reverted
```
Comment (yuvipanda) [inline config/clusters/2i2c-aws-us/cosmicds.values.yaml:85]: I've assigned https://github.com/2i2c-org/meta/issues/681 to myself to track this down. The rest of the PR is good to go, but let's back out the cosmicds change?

Comment (consideRatio) [reply config/clusters/2i2c-aws-us/cosmicds.values.yaml:85]: This change doesn't revoke the public access for CosmicDS, it just removes google and microsoft auth as identity providers - something that a comment (https://github.com/2i2c-org/infrastructure/issues/2128#issuecomment-1633128941) indicates was wanted.

Comment (yuvipanda) [reply config/clusters/2i2c-aws-us/cosmicds.values.yaml:85]: @consideRatio yep, but I'd like to not do that without checking with them explicitly. I've updated the meta ticket with an action I've taken to move this forward.

Comment (consideRatio) [reply config/clusters/2i2c-aws-us/cosmicds.values.yaml]: I reverted the commit removing google/microsoft as identity providers. You can do `git cherry-pick af74dc5` later to apply it again as its now part of the git history.

**Review by yuvipanda — APPROVED** [review]
Thanks!

**Comment by consideRatio** [comment]
Thank you @yuvipanda!!!

**Pull request merged**